library/pyproject.toml

```
@@ -27,7 +27,7 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-    "datumaro==1.10.0",
+    "datumaro==1.12.0",
```

```suggestion
    "datumaro~=1.12.0",
```
Let's keep flexibility on the patch version, if Datumaro releases any bugfix (e.g. 1.12.1)